backend/src/apiserver/template/v2_template.go

```diff
@@ -26,11 +27,13 @@ import (
 	scheduledworkflow "github.com/kubeflow/pipelines/backend/src/crd/pkg/apis/scheduledworkflow/v1beta1"
 	"github.com/kubeflow/pipelines/backend/src/v2/compiler/argocompiler"
 	"google.golang.org/protobuf/encoding/protojson"
+	goyaml "gopkg.in/yaml.v2"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 type V2Spec struct {
-	spec *pipelinespec.PipelineSpec
+	spec         *pipelinespec.PipelineSpec
+	platformSpec *pipelinespec.PlatformSpec
 }
```

Comment:
It seems that we have integration tests checking for the content of V2Spec, so they also need to be updated.

Reply:
Thank you, I just fixed it.